pagopa/io-app
# chore(Messaggi a valore legale): [IAMVL-37] Hide html selector when no html is present in the body

## Short description
When the MVL body has no `html` (as in _it is an empty string_), hide the `html` selector in the list.

## List of changes proposed in this pull request
- Also hide the separator since there is only one selectable item

## How to test

Use this branch of the io-dev-server to test it https://github.com/pagopa/io-dev-api-server/pull/121

Please check the video below for a demo.

https://user-images.githubusercontent.com/2094604/149807522-79540a5c-10a7-4122-8278-d95485103efc.mov
